generator/routing_index_generator.cpp

```diff
@@ -527,6 +527,7 @@ void FillWeights(string const & path, string const & mwmFile, string const & cou
     if (i % 10 == 0)
```

Comment:
```suggestion
    if (i++ % 10 == 0)
```
?

Reply:
I'm not sure about this suggestion because `Building leaps: 1/n waves passed` log message would be printed before any wave propagation happens. Perhaps we should move this whole condition at the bottom of the loop scope?